dashboard/app/assets/javascripts/dashboard.controller.js

```diff
@@ -0,0 +1,49 @@
+(function() {
+  'use strict';
+
+  angular.module('qat-dashboard').controller('DashboardController', [
+    'testExecutor',
+    function(testExecutor) {
+      var vm = this;
+
+      vm.testExecsMap = testExecutor.all;
+      vm.requestTestExec = requestTestExec;
+      vm.getDate = getDate;
+      vm.getStatusCssClass = getStatusCssClass;
```

Comment:
@christianvuerings Please check my explanation below. The relevant portion is discussed here: https://github.com/johnpapa/angular-styleguide#bindable-members-up-top